frontend/src/components/Curiosity/CuriosityPics.jsx

```diff
@@ -42,59 +49,105 @@ export default function CuriosityPics({ picList }) {
           <span className="police">Choose a camera :</span>
         </p>
         <select name="Camera" value={choosenValue} onChange={handleChange}>
-          <option value="">All</option>
-          <option value="FHAZ">Front Hazard Avoidance Camera - FHAZ</option>
-          <option value="RHAZ">Rear Hazard Avoidance Camera - RHAZ</option>
-          <option value="MAST">Mast Camera - MAST</option>
+          <option value="">All - {picList.length} photos</option>
+          <option value="FHAZ">
+            Front Hazard Avoidance Camera - FHAZ -{" "}
+            {picList.filter((pic) => pic.camera.name.includes("FHAZ")).length}{" "}
+            photos
+          </option>
+          <option value="RHAZ">
+            Rear Hazard Avoidance Camera - RHAZ -{" "}
+            {picList.filter((pic) => pic.camera.name.includes("RHAZ")).length}{" "}
+            photos
+          </option>
+          <option value="MAST">
+            Mast Camera - MAST -{" "}
+            {picList.filter((pic) => pic.camera.name.includes("MAST")).length}{" "}
+            photos
+          </option>
           <option value="CHEMCAM">
-            Chemistry and Camera Complex - CHEMCAM
+            Chemistry and Camera Complex - CHEMCAM -{" "}
+            {
+              picList.filter((pic) => pic.camera.name.includes("CHEMCAM"))
+                .length
+            }{" "}
+            photos
+          </option>
+          <option value="MAHLI">
+            Mars Hand Lens Imager - MAHLI -{" "}
+            {picList.filter((pic) => pic.camera.name.includes("MAHLI")).length}{" "}
+            photos
+          </option>
+          <option value="MARDI">
+            Mars Descent Imager - MARDI -{" "}
+            {picList.filter((pic) => pic.camera.name.includes("MARDI")).length}{" "}
+            photos
+          </option>
+          <option value="NAVCAM">
+            Navigation Camera - NAVCAM -{" "}
+            {picList.filter((pic) => pic.camera.name.includes("NAVCAM")).length}{" "}
+            photos
```

Comment:
Too many repetitions, that could be improved !

Reply:
New commit done to fix that